.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Dependabot Workflow for libseccomp
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+# based on this guide from GitHub:
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+    commit-message:
+      prefix: "RFE:"
```

Comment:
I think we may need a trailing space on the "RFE:" prefix, but I can tweak that during the merge ... and if I get it wrong, you can blame me ;)